### ISSUE TITLE
Create Release fails to create a release branch

### ISSUE THREAD

**Issue by AskeHolst** [microsoft/AL-Go#466]
When running the Create Release with Create Release Branch, there is an error regarding creating or updating workflows (see below).
I only get this in one repo, other repos with recent releases have worked fine.

The settings for both org and repo are permissive to actions:
![image](https://user-images.githubusercontent.com/8398485/230202986-a41e9894-b995-43c1-a1c3-691b3bc94d06.png)
There is a branch protection on release/* to require pull request, but we have the same rule on other repos where it works.
Is there some other setting missing?

```
Run $ErrorActionPreference = "STOP"
Switched to a new branch 'release/1.1'
[release/1.1 8186b33] Release branch release/1.1
To https://github.com/[REDACTED]/[REDACTED]
 ! [remote rejected] release/1.1 -> release/1.1 (refusing to allow a GitHub App to create or update workflow `.github/workflows/AddExistingAppOrTestApp.yaml` without `workflows` permission)
error: failed to push some refs to 'https://github.com/[REDACTED]/[REDACTED]'
Error: Process completed with exit code 1.
```

I can get around it my manually creating a branch from the release tag, so it's not a big problem.

**Comment by freddydk** [comment]
I saw this today as well - not sure why this comes some times and not always - will investigate

**Comment by AskeHolst** [comment]
We got it again and again in the same repo when I reported it, the other day it worked fine in the same repo on the same branch. Don't you just love intermittent problems :-)

**Comment by freddydk** [comment]
The fix I am working on for issue #555 probably would fix this as well.

**Comment by freddydk** [comment]
Even though this issue is oldest, I will close it as a duplicate
